includes/wpml/class-lesson-progress.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * File containing \Sensei\WPML\Lesson_Progress class.
+ *
+ * @package sensei
+ */
+
+namespace Sensei\WPML;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Class Lesson_Progress
+ *
+ * Compatibility code with WPML.
+ *
+ * @since $$next-version$$
+ *
+ * @internal
+ */
+class Lesson_Progress {
+	/**
+	 * Init hooks.
+	 */
+	public function init() {
+		add_filter( 'sensei_utils_user_completed_lesson_lesson_id', array( $this, 'translate_lesson_id' ), 10, 1 );
+		add_filter( 'sensei_lesson_progress_create_lesson_id', array( $this, 'translate_lesson_id' ), 10, 1 );
+		add_filter( 'sensei_lesson_progress_get_lesson_id', array( $this, 'translate_lesson_id' ), 10, 1 );
+		add_filter( 'sensei_lesson_progress_has_lesson_id', array( $this, 'translate_lesson_id' ), 10, 1 );
+		add_filter( 'sensei_lesson_progress_delete_for_lesson_lesson_id', array( $this, 'translate_lesson_id' ), 10, 1 );
+		add_filter( 'sensei_lesson_progress_find_lesson_id', array( $this, 'translate_lesson_id' ), 10, 1 );
```

Comment:
Maybe we can remove all the 10, 1 values as they are default?

Reply:
Fixed here: https://github.com/Automattic/sensei/pull/7492/commits/96814c963eb4a7cad88004d29e128ebe1c8a80da